util/src/main/scala/org/ensime/util/ensimefile.scala

```diff
@@ -77,6 +78,7 @@ package object ensimefile {
     override def readStringDirect()(implicit cs: Charset): String = withEntry(_.readString())
     override def uri: URI = URI.create(s"jar:${archive.jar.toUri}!${archive.entry}") // path is null (opaque)
     override def canon: ArchiveFile = ArchiveFile(archive.jar.canon, archive.entry)
+    override def toPath: Path = archive.jar
```

Comment:
this won't work... it's supposed to represent a file within an archive, not the archive itself.